src/fdk_rdf_parser/reference_data/extend_dataset.py

```diff
@@ -73,53 +72,43 @@ def extend_distributions(
             if ref_media_types:
                 fdk_formats: Set[MediaType] = set()
 
-                if dist.format:
-                    extended_formats = []
-                    for fmt in dist.format:
-                        ref_media_type = ref_media_types.get(fmt)
+                if dist.fdkFormat:
+                    skos_code_formats = []
+                    for fmt in dist.fdkFormat:
+                        ref_media_type = (
+                            ref_media_types.get(fmt.code) if fmt.code else None
+                        )
                         if ref_media_type:
-                            extended_formats.append(
-                                map_media_type_to_skos_code(ref_media_type)
+                            skos_code_formats.append(
+                                SkosCode(
+                                    uri=fmt.code,
+                                    code=ref_media_type.code,
+                                    prefLabel={"nb": ref_media_type.code}
```

Comment:
Det ble vel diskutert, men er det ikke ryddigere å ikke sette verdi for prefLabel heller enn å duplisere verdien?

Reply:
rimelig sikker på at dette SkosCode-greiene kun blir brukt for å finne hvilke ikoner som skal vises på detaljer-siden

Det skal fjernes etterhvert, men er det ikke tryggest å bare beholde enellerannen verdi der frem til vi fjernet SkosCode-feltene helt?